framework/src/executioners/Eigenvalue.C

```diff
@@ -67,8 +67,6 @@ Eigenvalue::validParams()
   // If Newton and Inverse Power is combined in SLEPc side
   params.addPrivateParam<bool>("_newton_inverse_power", false);
 
-  params.addParam<Real>("time", 0.0, "System time");
```

Comment:
Not removing this parameter because I noticed this line has been duplicated. Just removing the duplication here.